CHANGELOG.asciidoc

```diff
@@ -77,6 +77,8 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Fixed the location of the modules.d dir in Deb and RPM packages. {issue}8104[8104]
 - Add docker diskio stats on Windows. {issue}6815[6815] {pull}8126[8126]
 - Fix incorrect type conversion of average response time in Haproxy dashboards {pull}8404[8404]
+- Fix dropwizard module parsing of metric names. {issue}8365[8365]
+- Fix dropwizard module parsing of metric names. {issue}8365[8365] {pull}6385[8385]
```

Comment:
Umm, this is duplicated now :thinking: only second line would be needed.

Reply:
Ups, good catch.